### PR TITLE
[tlul] Change TLUL back to D1V1

### PR DESCRIPTION
**Pull request by weicaiyang**
New features and bug-fixing are being added to TLUL, so
1. Change version from 0.5 to 1.0
2. Change stage from L2D3V3 to L1D1V1
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

**Comment by eunchan** [comment]
Could you create new entry not modifying the signed off list?

**Comment by eunchan** [comment]
@weicaiyang As PR #1009 was merged, could you please revise the project doc to have multiple versions in the `revisions` field? It could be similar with this:

```hjson
{
    name:               "tlul",
    revisions: [
      {
        version:            "0.5",
        life_stage:         "L2",
        design_stage:       "D3",
        verification_stage: "V3",
        commit_id:          "f723f327bae4c70a14409ab260ddcb7856be2ebd",
        notes:              "the target for this entry is the autogenerated xbar_main within top_earlgrey."
      }
      {
        version:            "1.0",
        life_stage:         "L1",
        design_stage:       "D1",
        verification_stage: "V1",
        commit_id: "",
        notes:              "",
      }
    ]
}
```

**Comment by weicaiyang** [comment]
> @weicaiyang As PR #1009 was merged, could you please revise the project doc to have multiple versions in the `revisions` field? It could be similar with this:

Updated, please take a look

File: hw/ip/tlul/data/tlul.prj.hjson [hunk 1]
```diff
@@ -4,13 +4,23 @@
...
+        verification_stage: "V3",
+        commit_id:          "f723f327bae4c70a14409ab260ddcb7856be2ebd",
+        notes:              "the target for this entry is the autogenerated xbar_main within top_earlgrey."
+      }
+      {
+        version:            "1.0",
+        life_stage:         "L1",
+        design_stage:       "D1",
+        verification_stage: "V1",
+        commit_id:          "0adb90e886ee9d9a2e931ff789130d0229923e89",
```
Comment (eunchan) [inline hw/ip/tlul/data/tlul.prj.hjson]: I cannot find the commit.
https://github.com/lowrisc/opentitan/tree/0adb90e886ee9d9a2e931ff789130d0229923e89

The commit ID should be visible before this PR is created.

Comment (weicaiyang) [reply hw/ip/tlul/data/tlul.prj.hjson]: Updated, PTAL

**Pull request merged**